boards/st/stm32mp135f_dk/Kconfig.defconfig

```diff
@@ -12,4 +12,7 @@ config GPIO_MCP230XX_INIT_PRIORITY
 
 endif # GPIO_MCP230XX
 
+configdefault NET_L2_ETHERNET
+	default y if NETWORKING
```

Comment:
Why not doing as other boards:
```
if NETWORKING

config NET_L2_ETHERNET
	default y

endif # NETWORKING
```

Reply:
This update is related to @maass-hamburg comment:  https://github.com/zephyrproject-rtos/zephyr/pull/96134#discussion_r2440383738 

@erwango , @maass-hamburg : Could you please clarify the good syntax here?

Reply:
coming in https://github.com/zephyrproject-rtos/zephyr/pull/98006